### ISSUE TITLE
GHA Windows TAO builds failing, possibly due to Perl

### ISSUE THREAD

**Issue by iguessthislldo** [DOCGroup/ACE_TAO#2148]
See https://github.com/OpenDDS/OpenDDS/discussions/4313#discussioncomment-7355551 for details.

**Comment by jwillemsen** [comment]
This is very likely a MPC issue

**Comment by iguessthislldo** [comment]
Yes, I noted that when mentioning this in the discussion. I wasn't sure we proved that yet though and TAO is what's ultimately having the issue. We wanted to mention this in the OpenDDS release today and needed an issue to link to.
